### DISCUSSION TITLE
/ref/baota.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#9] [Announcements]
# /ref/baota.html

VanBlog 的官方网站

https://vanblog.mereith.com/ref/baota.html

**Comment by NiuXiaoGuang** [comment]
我部署报错，提示“此端口 [80] 已经被其他模板使用！”

**Reply by Mereithhh** [reply]
换个端口就好了。。我一会在文档补上换端口的教程。

**Comment by haowee** [comment]
为啥我的宝塔面板没有docker显示啊

**Reply by haowee** [reply]
必须升级到宝塔版本必须升级到7.9.4以上才行
